pkgs/applications/editors/vim/plugins/blink-cmp/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
```

Comment:
This appears to be unused.

```suggestion
```

Aside from that, this looks good to me

Reply:
Removed it, thanks!